src/browser/ui/dom/SVGDOMPropertyConfig.js

```diff
@@ -59,10 +66,34 @@ var SVGDOMPropertyConfig = {
     x1: MUST_USE_ATTRIBUTE,
     x2: MUST_USE_ATTRIBUTE,
     x: MUST_USE_ATTRIBUTE,
+    xlinkActuate: MUST_USE_ATTRIBUTE | HAS_NAMESPACE,
```

Comment:
Seems like this HAS_NAMESPACE is redundant with the NamespaceProperties dict below -- is that right? If so, better to have only one of them.